spip_svn_loader

```diff
@@ -272,7 +288,7 @@ then
         _BRANCHE=$SVN_SPIP_TAGS"/"$brancheREPONSE
     fi
 
-    WC=`find . -type d -maxdepth 1 -name .svn -exec echo {} \;`
+    WC=`find . -maxdepth 1 -type d -name .svn -exec echo {} \;`
```

Comment:
Tu m'en demandes trop, je ne sais pas faire un PR juste pour une ligne ^^

On Thu, Mar 17, 2016 at 8:31 PM, James <notifications@github.com> wrote:

> In spip_svn_loader
> <https://github.com/JamesRezo/spip_svn_loader/pull/9#discussion_r56564270>
> :
>
> > @@ -272,7 +288,7 @@ then
> >          _BRANCHE=$SVN_SPIP_TAGS"/"$brancheREPONSE
> >      fi
> >
> > -    WC=`find . -type d -maxdepth 1 -name .svn -exec echo {} \;`
> > +    WC=`find . -maxdepth 1 -type d -name .svn -exec echo {} \;`
>
> OK pour ça, peux-tu faire une PR spécifique pour cette ligne stp ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/JamesRezo/spip_svn_loader/pull/9/files/d04d19fc515865f89acff812e24022af14163c71#r56564270>
>